common/types/activation.go

```diff
@@ -180,18 +162,13 @@ func (atxh *ActivationTxHeader) TickHeight() uint64 {
 	return atxh.baseTickHeight + atxh.tickCount
 }
 
-// Verify sets field extract after verification.
-func (atxh *ActivationTxHeader) Verify(baseTickHeight, tickCount uint64) {
-	atxh.verified = true
-	atxh.baseTickHeight = baseTickHeight
-	atxh.tickCount = tickCount
-}
-
 // NIPostChallenge is the set of fields that's serialized, hashed and submitted to the PoET service to be included in the
 // PoET membership proof. It includes ATX sequence number, the previous ATX's ID (for all but the first in the sequence),
 // the intended publication layer ID, the PoET's start and end ticks, the positioning ATX's ID and for
 // the first ATX in the sequence also the commitment Merkle root.
 type NIPostChallenge struct {
+	// Sequence number counts the number of ancestors of the ATX. It sequentially increases for each ATX in the chain.
+	// It can be used in proving to other nodes that a malicious party created more ATXs than they would are allowed to.
```

Comment:
rephrase to "Two ATXs with the same sequence number from the same miner can be used as the proof of malfeasance against that miner."